src/routes/renderRoutes.js

```diff
@@ -1,41 +1,36 @@
 const db = require("../../sequelize/db");
 const es = require("../../database/elasticsearch/elastic");
 const pg = require("../../database/postgres/pg");
-
+const arangoModule = require("../../database/arango/arango");
+require("dotenv").config();
 const routes = {};
 
+// This is the 'host' url for a person's database credentials
+const dbHost = {
+  Postgres: "learndatabases.dev",
+  Elasticsearch: "elastic.learndatabases.dev",
+  Arango: process.env.ARANGO_URL,
+};
```

Comment:
Shouldn't it be `process.env.HOST` for postgres/sequelize and `process.env.ES_HOST` for elastic?

Reply:
You are right -- I simply copy and pasted the links used for `dbHost` that the last person that touched this file used. It should be `process.env.ES_HOST` for elastic, and the testing files should change to reflect that too. For Postgres, I am not sure. It seems we just use `127.0.0.1` as our production `HOST` env variable.